gears/gears-compose/README.md

```diff
@@ -0,0 +1,46 @@
+# compose <GitHub path="RedMadRobot/gears-android/tree/main/gears/compose"/>
+[![Version](https://img.shields.io/maven-central/v/com.redmadrobot.gears/gears-compose?style=flat-square)][mavenCentral]
+[![License](https://img.shields.io/github/license/RedMadRobot/gears-android?style=flat-square)][license]
+
+---
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+A set of gears for compose.
+
+## Installation
+
+Add the dependency:
+```groovy
+repositories {
+    mavenCentral()
+    google()
+}
+
+dependencies {
+    implementation("com.redmadrobot.gears:gears-compose:<version>")
+}
+```
+
+The library was developed and tested using `compose:1.6.6`, and it includes compose-ui and compose-runtime of this version transitively as dependencies in your project.
```

Comment:
I'd suggest listing dependencies like it is done in [encrypted-datastore](https://github.com/osipxd/encrypted-datastore)

<img width="249" alt="image" src="https://github.com/RedMadRobot/gears-android/assets/3684355/f5ba3fe6-b76d-4ccb-9d9a-3adae618f31d">

Reply:
![image](https://github.com/RedMadRobot/gears-android/assets/18189767/c9edd348-90f1-42e5-b379-05f8e6436725)